src/main/java/net/pms/service/ProcessManager.java

```diff
@@ -840,9 +840,11 @@ protected void destroyProcess(@Nullable Process process) {
 			if (process == null) {
 				return;
 			}
-			IOUtils.closeQuietly(process.getInputStream());
-			IOUtils.closeQuietly(process.getErrorStream());
-			IOUtils.closeQuietly(process.getOutputStream());
+
+			IOUtils.closeSilently(process.getInputStream());
```

Comment:
Why do we want `closeSilently` here, but in the other files we changed to `close`? It doesn't need to change, I just want to understand

Reply:
@SubJunk frankly I didn't find the way how to use try-with-resources instead. 😞

Reply:
Ah cool, I think you can just separate them by semicolons or something, I'll try to do it

Reply:
Oh I see, it is all split up into different methods, so yeah this is probably the best way unless we refactor it. Nice work